io.spectra.sudoku_solver.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id" : "io.tachyon.sudoku_solver",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "sudoku-solver",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+              {
+                "type": "git",
+                "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                "tag": "v0.6.0"
+              }
+            ]
+        },  
+        {
+            "name" : "sudoku-solver",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dbuildtype=release"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://gitlab.com/cyberphantom52/sudoku-solver/uploads/12631eee2356af905977ac603642c50e/sudoku-solver-1.0.0.tar.xz",
```

Comment:
it's very bad taste to retag.

Reply:
Can you elaborate what is meant by retagging, please.

Reply:
same version, different tarball.

Reply:
Oh!, i'll keep that in mind, i'll do different releases from next time.

Reply:
I've released the version with fixed application-id separately as v1.0.1 in https://github.com/cyberphantom52/flathub/commit/57c5698bdc252f6c193900d7533c3427db836948

From now on any patches will be treated as incremental versions.